spec/omni_exchange/providers/xe_spec.rb

```diff
@@ -73,13 +73,13 @@
     it 'returns the exchange rates from the date specified' do
       VCR.use_cassette('omni_exchange/xe_historic_rate') do
         rate = subject.get_historic_rate(base_currency: 'USD',
-                                         target_currencies: %w[EUR JPY KRW],
-                                         date: Date.new(2018, 1, 1))
+                                         target_currencies: %w[JPY KRW EUR],
+                                         date: Date.new(2017, 1, 1))
```

Comment:
We were using in several places the cassette for `xe_historic_rate`. This included 2 calls, for `2017` and `2018`.
As I replaced the cassette using the new url, I decided to stick with only one of the dates as I see no need to check for more. We only need to confirm that we get values back given the provided information. The order here was also relevant for the cassette to be re used